ros/angel_system_nodes/angel_system_nodes/activity_classification/activity_classifier_tcn.py

```diff
@@ -161,16 +156,16 @@ def __init__(self):
                 (PARAM_DEBUG_FILE, ""),
             ],
         )
-        self._img_ts_topic = param_values[PARAM_IMG_TS_TOPIC]
+        self._img_md_topic = param_values[PARAM_IMG_MD_TOPIC]
         self._det_topic = param_values[PARAM_DET_TOPIC]
 
         self._pose_topic = param_values[PARAM_POSE_TOPIC]
         self._pose_repeat_rate = param_values[PARAM_POSE_REPEAT_RATE]
 
         self._act_topic = param_values[PARAM_ACT_TOPIC]
         self._act_config = load_activity_label_set(param_values[PARAM_ACT_CONFIG_FILE])
-        self._img_pix_width = int(param_values[PARAM_IMAGE_PIX_WIDTH])
-        self._img_pix_height = int(param_values[PARAM_IMAGE_PIX_HEIGHT])
+        self._img_pix_width = 1280  # default size - replaced by metadata topic
```

Comment:
I hesitate to have a default value here that could lead to sneaky issues when we assume downstream the metadata topic has replace these width & height values with those of the video frame

Reply:
I would second this. It should probably be the case that the windowing functionality is updated to store `CameraInfo` instances instead of `Time` so that the window slices come with these metadata-carrying objects to assign image sizes to `FrameData` on a per-frame basis (despite not particularly expecting this to change per-frame).